src/modules/transaction/components/TransactionRow/index.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { withTranslation } from 'react-i18next';
```

Comment:
I don't want to touch the transaction row since its out of scope for this PR. I think this should be addressed in this issue https://github.com/LiskHQ/lisk-desktop/issues/4423